stakingModel/contracts/GoodMarketMaker.sol

```diff
@@ -54,6 +83,7 @@ contract GoodMarketMaker is BancorFormula, DSMath, SchemeGuard {
         onlyAvatar
     {
         reserveRatioDailyExpansion = rdiv(_nom, _denom);
+        emit ReserveRatioUpdated(msg.sender, _nom, _denom);
```

Comment:
description isnt correct. you changed it so it no longer defaults to 20%
verify other methods comments are correct